### DISCUSSION TITLE
getting error while pushing the code into Git

### DISCUSSION THREAD

**Discussion by srinivasgunthapally** [community/community#44758] [Codespaces]
### Select Topic Area

Product Feedback

### Body

while my code is pushing into git repository already exit not getting into the code into repository how to resolve the issue

**Comment by msuliq** [comment]
What does the error message say? Could you please provide more context

**Reply by srinivasgunthapally** [reply]
i have attached the file while pushing the code
![Screenshot (32)](https://user-images.githubusercontent.com/117246347/213521224-5b19051c-1ca8-4e33-b35d-ab7af80c1d1b.png)

**Comment by srinivasgunthapally** [comment]
![Screenshot (32)](https://user-images.githubusercontent.com/117246347/213521086-dff90fa8-c488-4c48-b45e-1a3fe0c34681.png)

**Comment by msuliq** [comment]
I see that the main error is `git remote origin already exists`
You could try resetting the origin again by typing the following command in your terminal:
`git remote set-url origin <your repo here...>`

Another solution would be to remove previously set git remote and set it again:
`git remote rm origin`
And then use the command
`git remote add origin  <your repo here...>`
Let me know if these help or don't help

**Reply by srinivasgunthapally** [reply]
getting same issue already exits could eloborate the process

**Reply by msuliq** [reply]
Ok, I will need some more context on this please.
Could you please tell if this is a completely new and empty repository? If yes, can you please copy the code from repository, it should look like below:
```
…or create a new repository on the command line
echo "# asdf" >> README.md
git init
git add README.md
git commit -m "first commit"
git branch -M main
git remote add origin git@github.com:msuliq/asdf.git
git push -u origin main
…or push an existing repository from the command line
git remote add origin git@github.com:msuliq/asdf.git
git branch -M main
git push -u origin main
```

**Reply by srinivasgunthapally** [reply]
how to remove existing repository getting same issue am taking new repositories but getting error as a already existing repository please help!
